src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -347,72 +352,54 @@ const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
-                </DialogTitle>
-                <DialogContent>
-                    <TextFieldInput
-                        label={'nameProperty'}
-                        value={studyName}
-                        setValue={(newValue) => setValue(STUDY_NAME, newValue)}
-                        error={studyNameErrorMessage}
-                        autoFocus
-                        adornment={studyNameAdornment}
-                        setValueHasChanged={setStudyNameChanged}
-                    />
-                    <TextFieldInput
-                        label={'descriptionProperty'}
-                        value={description}
-                        setValue={(newValue) => setValue(DESCRIPTION, newValue)}
-                    />
-                    {providedExistingCase ? (
-                        <DirectorySelect types={[ElementType.DIRECTORY]} />
-                    ) : (
-                        <UploadNewCase
-                            caseFile={caseFile}
-                            caseFileLoading={caseFileLoading}
-                            handleCaseFileUpload={handleCaseFileUpload}
-                        />
-                    )}
-                    <ImportParametersSection
-                        onChange={handleParamsChange}
-                        currentParameters={currentParameters}
-                        formatWithParameters={formattedCaseParameters}
-                    />
-                    <Grid pt={1}>
-                        {!!apiCallErrorMessage && (
-                            <Alert severity="error">
-                                {apiCallErrorMessage}
-                            </Alert>
-                        )}
-                        {caseFileErrorMessage && (
-                            <Alert severity="error">
-                                {caseFileErrorMessage}
-                            </Alert>
-                        )}
-                    </Grid>
-                </DialogContent>
-                <DialogActions>
-                    <Button onClick={handleCancelStudyCreation}>
-                        <FormattedMessage id="cancel" />
-                    </Button>
-                    <Button
-                        onClick={handleSubmit(handleCreateNewStudy)}
-                        disabled={!isCreationAllowed}
-                        variant="outlined"
-                    >
-                        <FormattedMessage id="validate" />
-                    </Button>
-                </DialogActions>
-            </Dialog>
-        </FormProvider>
+        <CustomMuiDialog
+            titleId={'createNewStudy'}
+            formSchema={schema}
+            formMethods={createStudyFormMethods}
+            removeOptional={true}
+            open={open}
+            onClose={onClose}
+            onSave={handleCreateNewStudy}
+            onCancel={handleCancelStudyCreation}
+            disabledSave={!isCreationAllowed}
+        >
+            <TextInput
```

Comment:
Same as `CreateCaseDialog`

Reply:
done, thank you

Reply:
The TS was already created (2368)

Reply:
I took the TS. I will start after this TS merge